src/opt/publisher/Gemfile.lock

```diff
@@ -175,26 +170,23 @@ GEM
     tilt (2.0.11)
     toml (0.3.0)
       parslet (>= 1.8.0, < 3.0.0)
-    typhoeus (1.4.1)
-      ethon (>= 0.9.0)
     tzinfo (2.0.6)
       concurrent-ruby (~> 1.0)
     uglifier (3.2.0)
       execjs (>= 0.3.0, < 3)
     webrick (1.8.1)
-    yell (2.2.2)
-    zeitwerk (2.6.12)
 
 PLATFORMS
-  ruby
+  aarch64-linux
+  arm-linux
+  arm64-darwin
+  x86-linux
+  x86_64-darwin
+  x86_64-linux
+  x86_64-linux-musl
```

Comment:
Same question about requirements. It's great that we have options, but why?

Reply:
This is generated programatically